Switch final spawner to KEEN way with callback delegate

dorimanx opened the pull request:
this is more simple and fast way to spawn grids as in keen code.

Commit:
Switch final spawner to KEEN way with callback delegate

LordTylus commented:
As written on discord this PR breaks fixing ships in gravity, as it now adds them immediately after they have been loaded again.

instead of waiting till the rest are finished also. resulting in disconnection of subgrids such as rotor heads which fall to the ground before the main grid is even loaded.

Commit:
Correct Main Spawner code.

* now will disable physics to prevent from subparts to move.
* set gravity to 0 for grid during spawn.
* load first the main grid, and then subgrids.

Tested in gravity with many subgrids and huge grid, and nothing was disconnected.
